tests/functional/base_test.go

```diff
@@ -262,3 +297,55 @@ func SimulateDaemonsetNumberReadyWithPods(name types.NamespacedName, networkIPs
 
 	logger.Info("Simulated daemonset success", "on", name)
 }
+
+func GetDNSData(name types.NamespacedName) *infranetworkv1.DNSData {
+	dns := &infranetworkv1.DNSData{}
+	Eventually(func(g Gomega) {
+		g.Expect(k8sClient.Get(ctx, name, dns)).Should(Succeed())
+	}).Should(Succeed())
+
+	return dns
+}
+
+func GetDNSDataList(name types.NamespacedName, labelSelector string) *infranetworkv1.DNSDataList {
+	dnsList := &infranetworkv1.DNSDataList{}
+	dnsListOpts := client.ListOptions{
+		Namespace: name.Namespace,
+	}
+	client.MatchingLabels{
+		"service": labelSelector,
+	}.ApplyToList(&dnsListOpts)
+	Eventually(func(g Gomega) {
+		g.Expect(k8sClient.List(ctx, dnsList, &dnsListOpts)).Should(Succeed())
+	}).Should(Succeed())
+
+	return dnsList
+}
+
+func GetDNSDataHostnameIP(dnsDataName string, namespace string, dnsHostname string) string {
+	dnsEntry := GetDNSData(types.NamespacedName{Name: dnsDataName, Namespace: namespace})
+	// DNSData contains Hosts (an array), this array will contain
+	// ovsdbserver-sb-0.ns.svc and ovsdbserver-sb.ns.svc, need to get
```

Comment:
this is no longer correct, right? you removed -%d records?

Reply:
True, forgot to remove comment, good catch